content/search/results.js

```diff
@@ -51,6 +51,18 @@
     return $results;
   };
 
+  var debounce = function debounce (func, delay) {
+    let timeout;
+    return function (...args) {
+      clearTimeout(timeout);
+      timeout = setTimeout(() => func.apply(this, args), delay);
+    };
+  };
+
+  var trackSearch = function trackSearch (query) {
+    window.posthog.capture('support-search', { query: $input.value });
```

Comment:
```suggestion
    window.posthog ||= { capture: console.log };
    window.posthog.capture('support-search', { query: $input.value });
```

Reply:
you could define posthog if non-existant and then you would get logging without an error locally

Reply:
I'll create something similar to what we have in dnsimple-app

Reply:
✅ Done